modules/ROOT/pages/deploying-to-multiple-environments.adoc

```diff
@@ -4,23 +4,33 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: deploy, deploying, cloudhub, on premises, on premise, environment
 
-Throughout the different phases in the development lifecycle, like QA, pre-production or production, you may need to configure your application differently as server names, credentials, HTTP ports, and other similar parameters that can vary.
+You can configure your Mule application to automatically load property files based on the development environment (QA, preproduction, production, etc) you are working on, making it easier to deploy your app both on-premises and in the cloud. This configuration requires you to complete the following tasks:
 
-You can configure your Mule application to facilitate deployment to one of many different environments, both on-premises and in the cloud, by using different configuration files for your development environments (prod, qa, etc) so your application dynamically loads property files based on the environment you are working on. To do so, you must complete the following macro steps:
+. Create a properties file for each development environment in your application.
 
-. Create a properties file for each environment in your application.
+. Configure a properties placeholder in your app to look for the environment upon launch.
 
-. Configure a properties placeholder in your application to look for the deployment environment upon launch.
-
-. Configure an environment variable to point to a specific environment during application deployment.
+. Set an environment variable to point to a specific development environment during application deployment.
 
 
 Consider externalizing other aspects of your configuration, like time-out values, polling frequencies, etc. Even if they don't vary between environments. This will facilitate tuning and experimenting as the whole Mule application would become configurable through a single properties file.
 
 link:../assets/attachments/multiple-envs-example.jar[Download Example Studio Application]
 
 
-== Basic Anatomy
+== Basic Anatomy of an Environment Property Configuration
+
+A few things to consider before creating and configuring environment properties:
+
+* Use a consistent naming strategy for your properties files and make them unique across applications. This will be easier for re-using across teams.
+* A good strategy is to use the application name or ID in the default and override properties file names.
```

Comment:
"...in the default" sounds odd in this sentence, maybe it can be changed to something like: 
"A good strategy is to add the application name or ID to the properties file names, for example `myApp-qa.properties`"